src/linker/ErrorAndWarningCodes.md

```diff
@@ -1,4 +1,139 @@
 # IL Linker Error Codes
+
+### IL1001
+```
+Failed to process XML substitution: 'XML document location'. Feature 'feature' does not specify a "featurevalue" attribute
```

Comment:
https://github.com/mono/linker/pull/1154 changed the format to put the error text on the same line as the error code. I personally prefer the style you have here since it avoids long lines in the heading, but I don't feel strongly about - either way, we should be consistent.

Reply:
@marek-safar is there a particular reason you prefer them to be on the same line?

Reply:
Oh, and https://github.com/mono/linker/pull/1159/files is further tweaking the format.

Reply:
Yep, designer review ;-)